dashboard/app/models/level.rb

```diff
@@ -329,7 +329,7 @@ def channel_backed?
   end
 
   def key
-    if level_num == 'custom'
+    if level_num == 'custom' || level_num.nil?
```

Comment:
Needed for this PR.  Without this change `key` is incorrect for DSL levels.  E.g.:

```diff
-level 'VideoGamesCoordinatePlanes'
+level 'blockly:Unplugged:'

-level 'Coordinates Multi 1'
+level 'blockly:Multi:'
```